src/Data/Cardinality.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace LORIS\Data;
+
+/**
+ * Cardinality represents the number of data points which
+ * apply to the scope of a data type.
+ *
+ * Since the Cardinality class represents an enumeration, the
+ * class is final.
+ *
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+final class Cardinality implements \JsonSerializable
+{
+    // Valid cardinality types for data to apply to.
+
+    /**
+     * A Unique Cardinality signifies that the data is unique
+     * across the scope. Examples of unique data are CandID
+     * for the candidate scope or VisitLabel for the Session
```

Comment:
if you mean the SQL sessions table that's not entirely correct. if you mean the visits table (ie list of possible visits independent of any candidate) then it is correct but confusing here

Reply:
Edit: my comment is incorrect. you dont need to answer it